build/root/Makefile.generated_files

```diff
@@ -182,7 +182,7 @@ $(META_DIR)/$(DEEPCOPY_GEN).todo: $(DEEPCOPY_FILES)
 
 $(DEEPCOPY_FILES): $(DEEPCOPY_GEN)
 	if [[ "$(DBG_CODEGEN)" == 1 ]]; then        \
-	    echo "DBG: deepcopy needed $(@D): $?";  \
+	    echo "DBG: deepcopy needed $(@D):";  \
```

Comment:
How about `"DBG: deepcopy needed $(@D): $(wordlist 1,3,$?)"` so we list a few files (some comment for all 3)

Reply:
Actually, you know what - skip this.  We're going to `ls` anyway.

Reply:
@thockin, Yes, you are right